src/data/repositories/DataSetD2Repository.ts

```diff
@@ -333,73 +330,73 @@ export class DataSetD2Repository implements DataSetRepository {
         const allCategoryCombos = dataSets.flatMap(dataSet => {
             return this.buildCategoryCombinationsByDataElements(dataSet)
                 .filter(cc => !cc.existing)
-                .map(cc => {
-                    const { categoryCombo } = cc;
-                    return categoryCombo;
-                });
+                .map(cc => cc.categoryCombo);
```

Comment:
not important, but this "cc" is, in fact, a dataElement (DataElementWithCombination), right?